docs/methoddocs/high_risk_employee.md

```diff
@@ -0,0 +1,15 @@
+# High Risk Employee
+
+```eval_rst
+.. autoclass:: py42.modules.detectionlists.DetectionListsModule
```

Comment:
There is also a departing employee client that is a detection list but it is on its .md file. Do we want to include that one here and just name the file `detectionlists.md`?

Reply:
Changed.

Reply:
Don't forget to rename it to `detectionlists.md` if we go this route

Reply:
After making this change, facing below error while generating docs. Not able to figure out the reason. 

I have tried, `make clean ` too.

```
WARNING: autodoc: failed to import class 'high_risk_employee.HighRiskEmployeeClient' from module 'py42.clients.detectionlists'; the following exception was raised:
cannot import name 'HighRiskEmployeeClient'
```

Reply:
@alanag13 @unparalleled-js : I am facing a weird error,  due to below line in `high_risk_employee.py` document generation is failing as mentioned in previous comment. 

`from py42.sdk.exceptions import Py42NotFoundError, Py42BadRequestError`

Any pointers on how to fix this?!! Can you please checkout this branch and confirm that you also are facing the same error.  

Also tested the same by creating a fresh virtual environment. If I remove this line document gets generated without any error.

Reply:
@kiran-chaudhary I will look into this, we have seen similar problems before.